### DISCUSSION TITLE
Morphir & Regnosys Pre-Tech Sprint Ideation

### DISCUSSION THREAD

**Discussion by mcleo-d** [finos/morphir#95] [Ideas]
This discussion has been created to house Morphir & Regnosys Pre-Tech Sprint ideas leading up to the April 2022 event.

**Comment by stephengoldbaum** [comment]
woohoo

**Comment by mcleo-d** [comment]
# Morphir / REGnosys Whiteboarding Output

The diagram below is the output of the Morphir / REGnosys whiteboarding session on Wednesday 23rd March at 5pm GMT / 1pm ET. 

![Morphir _ REGnosys Tech Sprint Sketch](https://user-images.githubusercontent.com/6029572/159772217-f27d5260-4719-4a5f-8bb9-d4287fc70559.png)

Feel free to comment below 👇🏻
